scenarios/kubernetes_e2e.py

```diff
@@ -417,7 +408,8 @@ def main(args):
     if args.skew:
         runner_args.append('--skew')
 
-    setup_extract(args.extract, mode, runner_args)
+    for ext in args.extract or []:
+        runner_args.append('--extract=%s' % ext)
```

Comment:
We have a unit test that validates that all kubernetes_e2e.py scenarios add an `--extract` flag. I don't think we need to enforce that here.

Soon I'd actually like to move all of these flags out of kubernetes_e2e.py and into config.json